domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/regex/DocumentTypeRegularExpressionValidationError.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.regex;
+
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationError;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.FieldTypeEntity;
+
+public class DocumentTypeRegularExpressionValidationError extends ValidationError {
```

Comment:
looking at the code of this one, if we change:
`Invalid regular expression for Document field...` to `Invalid regular expression for field...`
then this one is generic and could be used for all regex exceptions on any field type. So we could name it `RegularExpressionValidationError`

Reply:
done